README.md

```diff
@@ -170,5 +202,7 @@ Write what an example ERB file (aka view) might look like to list all the warrio
 
 Write your code here (**NOTE: syntax highlighting doesn't work for ERB in markdown files, so ignore the colors!**):
 ```html
-<!-- code here -->
+<ul><% @warriors.each do |warrior| %>
+  <li><%= warrior.name %></li>
+  <% end %></ul>
 ```
```

Comment:
for code maintainability and clarity i might move the `ul`'s to their own lines.